app/src/main/java/fr/free/nrw/commons/contributions/ContributionsActivity.java

```diff
@@ -268,24 +270,21 @@ public int getTotalMediaCount() {
     }
 
     private void setUploadCount() {
-        UploadCountClient uploadCountClient = new UploadCountClient();
         CommonsApplication application = CommonsApplication.getInstance();
-        ListenableFuture<Integer> future = uploadCountClient
-                .getUploadCount(application.getCurrentAccount().name);
-        Futures.addCallback(future, new FutureCallback<Integer>() {
-            @Override
-            public void onSuccess(Integer uploadCount) {
-                getSupportActionBar().setSubtitle(getResources()
-                        .getQuantityString(R.plurals.contributions_subtitle,
-                                uploadCount,
-                                uploadCount));
-            }
-
-            @Override
-            public void onFailure(@NonNull Throwable t) {
-                Timber.e(t, "Fetching upload count failed");
-            }
-        }, ExecutorUtils.uiExecutor());
+
+        compositeDisposable.add(
+                RxJava2Tasks.getUploadCount(application.getCurrentAccount().name)
```

Comment:
I'll leave it as it it now. Please feel free to move it around whenever makes sense (the day `ApacheHttpClientMediaWikiApi` disappears, or implements a Reactive interface?)